content/en/news/release-notes.md

```diff
@@ -6,6 +6,24 @@ weight: 1
 
 For additional information check our [sprint demo videos](https://www.youtube.com/channel/UCcm2NzDN_UCZKk2yYmOpc5w) and [blogs](https://medium.com/kialiproject).
 
+## 1.840
+Sprint Release: May 03, 2024
```

Comment:
I haven't noticed it until now but we don't release on Fridays anymore.. these release dates should be the Monday date (May 6)